quickstart.tfvars

```diff
@@ -0,0 +1,40 @@
+# Please refer to essbase.auto.tfvars.template for complete variables list and help.
```

Comment:
use spaces not tabs to ensure proper alignment

Reply:
remove OCIDs and replace with "<OCID>"

Reply:
Done